articles/virtual-desktop/create-service-principal-role-powershell.md

```diff
@@ -100,6 +72,34 @@ Here are the three credentials you should write down and the cmdlets you need to
     $svcPrincipal.AppId
     ```
 
+## Create a role assignment in Windows Virtual Desktop Preview
+
+Next you will create an RDS role assignment in Windows Virtual Desktop for the service principal, which will allow the service principal to sign in to Windows Virtual Desktop. Make sure to use an account that has permissions to create RDS role assignments.
```

Comment:
Next, you'll create an RDS role assignment in Windows Virtual Desktop for the service principal, which will allow the service principal to sign in to Windows Virtual Desktop. Make sure to use an account with permissions to create RDS role assignments.